google/auth/external_account.py

```diff
@@ -112,6 +114,7 @@ def __init__(
         self._audience = audience
         self._subject_token_type = subject_token_type
         self._token_url = token_url
+        self._token_info_url = token_info_url
```

Comment:
Not needed for now since we don't use it, but can you add a TODO to validate the token info URL?

Reply:
Done

Reply:
Actually lets validate this now

Reply:
Done